performance-es.md

```diff
@@ -183,3 +185,107 @@ de texto, pero el coste es que no será legible para un humano.
 Para la transferencia de datos, cambia a un protocol menos verboso, o
 mejora el API para aceptar consultas parciales. Por ejemplo, usa una consulta
 incremental en lugar de forzar a traer siempre el set de datos completo.
+
+### Cambiar de algoritmo
```

Comment:
`/s/Cambiar/Cambios`

Reply:
I am not sure about the translation for the section title. 

Your suggestion (`Cambios de algoritmo`) sounds good, but it does not mix well with the previous section about data that has been translated as `Modificar los datos`. I think it would be nice to find a common wording for the title of both sections, thoughts?

Reply:
That's a very good point, we should agree on a tense for all titles 👍

Reply:
The sections title is now `Modificar los algoritmos`. If we later find a better translation, we should updated both this title section and the one about modifying data.